Packs/ThycoticDSV/Integrations/ThycoticDSV/ThycoticDSV.yml

```diff
@@ -0,0 +1,66 @@
+category: Authentication
+commonfields:
+  id: ThycoticDSV
+  version: -1
+configuration:
+- display: Server URL (e.g. https://example.com)
+  name: url
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Client id for grant type
+  hidden: false
+  name: client_id
+  required: true
+  type: 0
+- display: Client secret for grant type
+  hidden: false
+  name: client_secret
+  required: true
+  type: 0
+description: Thycotic DevOps Secrets Vault is a high velocity vault that centralizes
+  secrets management, enforces access, and provides automated logging trails. This
+  cloud-based solution is platform agnostic and designed to replace hard-coded credentials
+  in applications, micro-services, DevOps tools, and robotic process automation. This
+  vault ensures IT, DevOps and Security teams the speed and agility needed to stay
+  competitive without sacrificing security.
```

Comment:
I suggest shortening this description, it seems like it will hurt visibility when it's as long as it is

Reply:
@guykeller 
Reduced description